WEAPON/RemoveWeaponComponentFromWeaponObject.md

```diff
@@ -5,11 +5,12 @@ ns: WEAPON
 
 ```c
 // 0xF7D82B0D66777611 0xA6E7ED3C
-void REMOVE_WEAPON_COMPONENT_FROM_WEAPON_OBJECT(Any p0, Any p1);
+void REMOVE_WEAPON_COMPONENT_FROM_WEAPON_OBJECT(Object weaponObject, Hash componentHash);
```

Comment:
Changing from `Any` to `Hash` will break C# scripts that don't use a `uint` there. Please change it to `int` instead to keep compatibility with C# scripts.

Reply:
```suggestion
void REMOVE_WEAPON_COMPONENT_FROM_WEAPON_OBJECT(Object weaponObject, int componentHash);
```